Tests/bash/test_imprivata_epic_core_wrapper_contracts.sh

```
@@ -0,0 +1,60 @@
+done
+
+for fragment in \
+  'Imprivata_OnesignEpicConnector_25.6.40154' \
+  'Imprivata_OneSignAgent_25.1.000.15' \
+  'Northwell_OneSign-SecureLock-Shortcut_1.0' \
+  'os.walk(source, followlinks=False)' \
+  '"entrypoint_file": "Install.cmd"' \
+  '"installer_type": "cmd"' \
+  'recursive_files_pinned_per_run_before_target_contact' \
+  '--package-set imprivata-epic-core' \
+  '--package-set-catalog "$MANIFEST"' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)